ui/app/common/obs/views/showObservation.html

```diff
@@ -12,7 +12,7 @@
         <span ng-if="observation.groupMembers.length > 1 && observation.formNamespace != null && observation.groupMembers[0].concept.conceptClass != 'Image'" class="testUnderPanel left">
             <label>{{::(observation.getDisplayValue())}}</label>
         </span>
-        <span ng-if="observation.groupMembers.length==0 && !configIsObservationForImages" class="testUnderPanel left">
+        <span ng-if="observation.groupMembers.length==0 && !configIsObservationForImages && observation.concept.conceptClass != 'Image'" class="testUnderPanel left">
```

Comment:
@rohit-yawalkar can you please check **configIsObservationForImages** or **observation.concept.conceptClass != 'Image'** check is redundant ?